src/Page/Api.elm

```diff
@@ -48,7 +48,19 @@ getApiServerUrl { clientUrl } =
 
 changelog : List News
 changelog =
-    [ { date = "9 novembre 2022"
+    [ { date = "14 novembre 2022"
+      , level = "minor"
+      , domains = [ "Textile" ]
+      , md =
+            """Un nouveau paramètre `dyeingMedieum` a été ajouté pour permettre de préciser le
```

Comment:
typo `dyeingMedium`

Reply:
Adressed in https://github.com/MTES-MCT/ecobalyse/pull/174/commits/c96549a02fb25477323960116be953c8ad0babb4